keps/sig-cli/2257-Kui/kep.yaml

```diff
@@ -0,0 +1,23 @@
+title: Kui Graphical Terminal Enhancements
+authors:
+  - "@starpit"
+  - "@myan9"
+  - "@paulcastro"
+owning-sig: sig-cli
+participating-sigs:
+  - sig-cli
+reviewers:
+  - "@pwittrock"
+  - "@soltysh"
+approvers:
+  - "@pwittrock"
+  - "@soltysh"
+editor: TBD
+creation-date: 2019-11-14
+last-updated: 2021-01-12
+status: implemented
```

Comment:
the KEP should be marked as implemented after graduating to GA - I guess it's not yet the case - if so it should still be in "implementable" state.